tests/pages/components/global-search.js

```diff
@@ -1,15 +1,22 @@
 import {
   clickable,
+  collection,
   create,
   fillable,
-  isVisible
+  isVisible,
+  text,
+  value
 } from 'ember-cli-page-object';
 
 const definition = {
   scope: '[data-test-global-search]',
   noResultsIsVisible: isVisible('.no-results'),
   input: fillable('input.global-search-input'),
-  clickIcon: clickable('[data-test-search-icon]')
+  clickIcon: clickable('[data-test-search-icon]'),
+  academicYear: value('select'),
```

Comment:
this is no specific enough. Please put a `[data-test-academic-year-filter]` attribute on the select element so it's clear what the test is using.

Reply:
Done.